events/event_handler.go

```diff
@@ -160,6 +161,18 @@ func (fh *eventHandler) handleStatusEvent(body []byte) error {
 		"TaskStatus": task.TaskStatus,
 	}).Info("Got StatusEvent")
 
+	app, err := fh.marathon.App(task.AppID)
```

Comment:
I'm not sure if I understand. Caching applications/tasks that are manages could help with performance (less calls to Marathon and Consul) and can be first step to fix https://github.com/allegro/marathon-consul/issues/170